.github/ISSUE_TEMPLATE/engineering-manager-onboarding.md

```diff
@@ -0,0 +1,67 @@
+
+     It will be your task to make this document into a substantially larger one, that draws from all the things you had to learn. You’re not on your own, your fellow peers will all be available to support you during this task. You got this!
+
+- [ ] The people-ops team should have signed you up for the various on-site manager trainings.
+
+- [ ] Your manager should have invited you to meetings relevant to managers.
+
+- [ ] Your manager should have scheduled 1:1s for you with the other engineering managers, including the CTO, and other leadership people.
+
+## Your first ~two months
+
+The goal of onboarding is to set you up for success as a new engineering manager at Artsy. To that end, your main objective during this time will be to gather knowledge. You will be given ample time to get acquainted with your new peers, our business, and our tech.
+
+_For now we’ll try to target roughly two months, as there are currently 5 product teams and sprints have a duration of 2 weeks. Whether or not this duration was too arbitrary is something we’ll have to learn._
+
+To help work on the main objective, we’ve tried to identify the following goals.
+
+### Get to know Artsy's team
+
+- [ ] Have 1:1s with each and every engineer (that includes tech-leads) to get to know the people, their interests, their north-star goals, and their perspectives on the businesses they contribute to.
+
+- [ ] While working with individual contributors and during your 1:1s, try to develop an understanding of where and how you want to add value as a manager.
+
+- [ ] Keep a continuous and open discussion going with your peer managers about opportunities for people you think you could manage and why. By the end of sprint rotation we should organically have arrived at a set of suggestions that we, as the entire engineering team, can _consider_.
+
+### Get to know Artsy's tech
+
+- [ ] Unlike individual contributors, who rotate teams for a few sprints to gain a general understanding of how we work as contributors, engineering managers are expected to rotate across _all_ the teams to gain the understanding needed to successfully support their reports in achieving their goals and growth. (For example, unblocking people by knowing how to achieve something or who/where to delegate to, facilitate discussions by understanding context and who to involve for a given topic; and how to set expectations for engineers, product, and business owners as to what is realistically achievable.) Tech-leads and PMs should have been made aware of your main objective and provide you with time to do so.
+
+- [ ] By the end of your sprint rotation, put together a short presentation that summarizes our stack and how different teams contribute to it. (Target audience TBD, although I _think_ it would be good to present it to everyone, both to contribute to presence _and_ so there’s more people that can provide feedback about potential misunderstandings.)
```

Comment:
This deliverable would rightly intimidate anyone, given the complexity and variety of the platform. Maybe that’s OK, but I’d consider ways of shrinking its scope.

Reply:
Do we keep Artsy technology stack somewhere besides the [blog post](http://artsy.github.io/blog/2017/04/14/artsy-technology-stack-2017/)? I think it'll will be useful to use this opportunity to keep updating it, e.g. the diagram (maybe still with teams in it) below, and encourage discussion with the whole team.

![screen shot 2018-10-30 at 10 32 18 am](https://user-images.githubusercontent.com/796573/47725529-1fce7d80-dc2f-11e8-9d74-8308354aabec.png)

Reply:
@dleve123 is interested in building something like ^ using a normal architecture diagramming tools (as opposed to something like the above that requires pretty deep sketch knowledge )

Reply:
I think original goal of this was just to have some outcome/deliverable from the mega-rotation. If it's not this specific thing, it could be TBD (or nothing at all).

Reply:
Re https://github.com/artsy/README/pull/95/files#r229330904: @joeyAghion and I are brainstorming an architecture diagram project for the Platform Practice

Generally, I'm pro having some concrete deliverable/session for someone new. IDK if it should be on the topic of the sprint rotation though (I do think that's pretty intimating), it could just be on a topic that the new manager wants to present upon (maybe something they see as a gap in Artsy).